packages/polaris-viz/CHANGELOG.md

```diff
@@ -5,7 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-<!-- ## Unreleased -->
+## Unreleased
+
+### Changed
+
+- Renamed `seriesColors.upToEight` to `seriesColors.limited`.
+- Removed `Olive` & `Orange` colors from `Light` theme.
```

Comment:
Should we also mention here that we updated some of the colors, for example INDIGO_10_SOLID  changed from `rgba(86, 42, 228, 1)` to `rgba(106, 66, 233, 1)` , or is it not worth mentioning? 😄

Reply:
I don't think so, i was just updating the variable names to match the figma. They're not actually exported out to consumers.